test/prob/test_fixture_cdf.hpp

```diff
@@ -485,18 +490,16 @@ class AgradCdfTestFixture : public ::testing::Test {
                                      const size_t N_REPEAT) {
     if (is_vec) {
       for (size_t i = 0; i < N_REPEAT; i++) {
-        EXPECT_FLOAT_EQ(
-            single_gradients[pos_single] * pow(single_cdf, N_REPEAT - 1),
-            multiple_gradients[pos_multiple])
+        EXPECT_NEAR(single_gradients[pos_single] / N_REPEAT,
+                    multiple_gradients[pos_multiple], 1e-7)
```

Comment:
I am confused by this change.

Reply:
First read: https://github.com/stan-dev/math/pull/2042/files#r485173044

That explains the pow. The other thing is the `/ N_REPEAT`. This has to do with the way `[a, a, a, a]` is constructed. As it is now, all those `a` s are actually different autodiff variables, and the `a` in `pow(cdf(a), N)` is the same, so each `a` in the vectorized case gets a 1/Nth of the adjoint value if that makes sense.

It's effectively done like this in the other files.